### ISSUE TITLE
Components with zero height before styling don't get displayed

### ISSUE THREAD

**Issue by raffomania** [NoDivide/astrum#62]
#### Tell us about the issue you are experiencing
Components that only have a height after being styled are not displayed. Instead, there is a message saying "This component is hidden at this resolution". 

#### What behaviour would you expect?
I would expect the component to be displayed.

#### How can this issue be replicated?
markup:
    
    <div class="box"></div>

style:

    .box {
        height: 100px;
        background-color: blue;
    }

#### Your environment

Platform: Arch Linux
OS Version: All packages up to date
Node Version: 6.3.1
NPM Version: 3.10.5
Astrum Version: 1.6.3

**Comment by RyanHavoc** [comment]
@raffomania Use the `disable_auto_sample_hiding` to explicitly set when a component should be shown e.g.:

```
"options": {
    "disable_auto_sample_hiding": {
        "show_on_mobile": true,
        "show_on_desktop": true
    }
}
```

Add this to your component in `data.json`. See [https://github.com/NoDivide/astrum#special-options](https://github.com/NoDivide/astrum#special-options) for more details.

**Comment by raffomania** [comment]
That's a possible workaround, but wouldn't it be more elegant to check the element visibility after the styles are loaded?

**Comment by RyanHavoc** [comment]
@raffomania That's what it should be doing. Your elements are rendered an Astrum checks their height to determine if they're visible are the current resolution. I'd need to see your Astrum instance to diagnose your problem further. Can you get it up somewhere accessible?

**Comment by raffomania** [comment]
Sadly, I can't make my instance accessible to you :( I've just tested my minimal example above, and it seems that the Problem only appears with my specific setup. I'll investigate and try to identify an exact cause.

**Comment by bloqhead** [comment]
I just ran into the same thing. The thing that solved it was to use `astrum delete [item]` and then I recreated it with a different name. I had a component named "hours" (like an "hours of operation" kind of deal) and I renamed it to "hours-of-operation". Not sure why but after I did that, the element rendered just fine and the message was gone.

Before I did this, I checked my `data.json` file and there were no special options applied to that element or anything. The file was exactly the same (with the exception of the renamed element).
